package.json

```diff
@@ -11,74 +11,88 @@
     "fetch"
   ],
   "packageManager": "pnpm@8.4.0",
-  "main": "./core/dist/index.js",
-  "module": "./core/dist/index.esm.js",
-  "types": "./core/dist/index.d.ts",
-  "typesVersions": {
-    "*": {
-      "infinite": [
-        "./infinite/dist/index.d.ts"
-      ],
-      "immutable": [
-        "./immutable/dist/index.d.ts"
-      ],
-      "mutation": [
-        "./mutation/dist/index.d.ts"
-      ],
-      "subscription": [
-        "./subscription/dist/index.d.ts"
-      ]
-    }
-  },
+  "main": "./dist/core/index.js",
+  "module": "./dist/core/index.esm.js",
+  "types": "./dist/core/index.d.ts",
   "exports": {
```

Comment:
We now have all exports in here, so could we remove all `package.json` files in subdirectories like `_internal/`, `immutable/`, `infinite/`, `mutation/` and `subscription/`?

Reply:
We kept them for `node10` resolution. can deprecate maybe in the next major version.